aptos-move/aptos-gas/src/move_stdlib.rs

```diff
@@ -4,24 +4,26 @@
 use move_stdlib::natives::GasParameters;
 
 crate::natives::define_gas_parameters_for_natives!(GasParameters, "move_stdlib", [
-    [.bcs.to_bytes.per_byte_serialized, "bcs.to_bytes.per_byte_serialized", 1],
-    [.bcs.to_bytes.failure, "bcs.to_bytes.failure", 10],
+    [.bcs.to_bytes.per_byte_serialized, "bcs.to_bytes.per_byte_serialized", 10],
+    [.bcs.to_bytes.failure, "bcs.to_bytes.failure", 1000],
```

Comment:
Why is this so expensive?

Reply:
Ah this is a good one. I guess I did it for no obvious reasons. 

The tricky thing is that bcs can fail, but the failure conditions are hardcoded in the bcs crate itself. :(
The failures only happens in the most extreme cases, like vector having over 2^32 values or values being nested too deep.
I honestly don't think we have a good story for this at all, be it for the bcs native or other parts of the Rust code that uses bcs...

Reply:
I think from_bytes/to_bytes are relatively common operations and should be cheaper if possible. Can we make them be closer to the cost of serializing/deserializing underneath

Reply:
Just to be clear, the "1000" here is the failure cost and as I mentioned above, a failure should be extremely rare...

Or is it that you think the unit cost is too high?

Reply:
Yeah I think it might be a bit high. But we can tune it later

Reply:
failure is impossible in bcs, imo